src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -184,6 +184,7 @@ def launch_fluent(
     cleanup_on_exit: bool = True,
     start_transcript: bool = True,
     show_gui: bool = None,
+    case_filepath: str = None,
```

Comment:
@mkundu1 @ajain-work @hpohekar @tejalprabhu01 

@prmukherj has merged his work into this branch: launching by case file path.